app/src/main/java/io/github/domi04151309/alwayson/alwayson/AlwaysOn.kt

```diff
@@ -131,17 +133,30 @@ class AlwaysOn : AppCompatActivity(), SensorEventListener {
     private var notificationGrid: RecyclerView? = null
     private val mNotificationReceiver = object : BroadcastReceiver() {
 
+        @SuppressLint("SetTextI18n")
         override fun onReceive(c: Context, intent: Intent) {
             val count = intent.getIntExtra("count", 0)
             if (aoNotifications) {
                 if (count == 0)
                     notifications!!.text = ""
-                else
-                    notifications!!.text = count.toString()
+                else {
+                    if (userTheme == "oneplus" && aoNotificationIcons)
+                        if (count > 3)
+                            notifications!!.text = "+" + (count - 3).toString()
+                        else
+                            notifications!!.text = ""
+                    else
+                        notifications!!.text = count.toString()
+                }
             }
 
             if (aoNotificationIcons) {
                 val itemArray: java.util.ArrayList<Icon> = intent.getParcelableArrayListExtra<Icon>("icons") ?: arrayListOf()
+                if (itemArray.size > 3) {
+                    for (i in itemArray.size - 1 downTo 3) {
+                        itemArray.removeAt(i)
+                    }
+                }
```

Comment:
This is a nice idea but currently the notification count and the notification icons are two seperate options.